doc/_extensions/zephyr/gh_utils.py

```diff
@@ -46,12 +47,33 @@
 from sphinx.application import Sphinx
 from sphinx.util.i18n import format_date
 
+ZEPHYR_BASE = Path(__file__).parents[3]
+SCRIPTS = ZEPHYR_BASE / "scripts"
+sys.path.insert(0, str(SCRIPTS))
+from get_maintainer import Maintainers
```

Comment:
I'm not aware of any coding guidelines we have for python, but shouldn't this `from` be with the others above?

Reply:
I don't think we have coding guidelines indeed, but I ran this through black formatter. 
I used this approach similarly to other extensions as well as `conf.py` -- since the import will only work after path has been tweaked (and `sys` imported), it didn't feel like a bad idea to do it this way